fuzz-testing/run.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+set -eux
+
+DIR="$(cd "$(dirname "$0")" && pwd)"
+PARENT_DIR="${DIR}/.."
+SPARK_MASTER="${SPARK_MASTER:-local[*]}"
+PROJECT_VERSION="$(mvn -q help:evaluate -Dexpression=project.version -DforceStdout)"
+COMET_SPARK_ARTIFACT_ID="$(mvn -f ${PARENT_DIR}/spark/pom.xml -q help:evaluate -Dexpression=project.artifactId -DforceStdout)"
+COMET_SPARK_JAR="${PARENT_DIR}/spark/target/${COMET_SPARK_ARTIFACT_ID}-${PROJECT_VERSION}.jar"
+COMET_FUZZ_ARTIFACT_ID="$(mvn -f ${DIR}/pom.xml -q help:evaluate -Dexpression=project.artifactId -DforceStdout)"
+COMET_FUZZ_JAR="${DIR}/target/${COMET_FUZZ_ARTIFACT_ID}-${PROJECT_VERSION}-jar-with-dependencies.jar"
+NUM_FILES="${NUM_FILES:-2}"
+NUM_ROWS="${NUM_ROWS:-200}"
+NUM_QUERIES="${NUM_QUERIES:-500}"
+
+if [ ! -f "${COMET_SPARK_JAR}" ]; then
+  echo "Building Comet Spark jar..."
+  make -f "${PARENT_DIR}/Makefile"
```

Comment:
```suggestion
  pushd "${PARENT_DIR}"
  make
  popd
```
I think you need to change the current working directory for `make`